src/java.base/share/classes/jdk/internal/util/StaticProperty.java

```diff
@@ -241,4 +246,29 @@ public static String fileEncoding() {
     public static String javaPropertiesDate() {
         return JAVA_PROPERTIES_DATE;
     }
+
+    /**
+     * Return the {@code sun.jnu.encoding} system property.
```

Comment:
This can be eliminated by changing the`@return` block tag below to `{@return the {@code sun.jnu.encoding} ...}`.

Reply:
Hello @naotoj .
I appreciate your comment.
I'd like to confirm one thing.
I applied following change
```
    /**
     * {@return the {@code sun.jnu.encoding} system property}
     *
     * <strong>{@link SecurityManager#checkPropertyAccess} is NOT checked
     * in this method. The caller of this method should take care to ensure
     * that the returned property is not made accessible to untrusted code.</strong>
     */
```

By javadoc command with -html5 option, above part was converted to
```
Returns the sun.jnu.encoding system property. SecurityManager.checkPropertyAccess(java.lang.String) 
is NOT checked in this method. The caller of this method should take care to ensure that the returned 
property is not made accessible to untrusted code.
```
The 1st word changed from **Return** to **Returns**.
Is it OK ?
And it seems `@return` is translated to Japanese on Japanese environment.

Reply:
> The 1st word changed from Return to Returns.
Is it OK ?

That's more of plain English to me. Maybe some natives can correct it if needed.

> And it seems @return is translated to Japanese on Japanese environment.

Yes. I believe that's what is supposed to work.